src/javascript/EditPanel/EditPanelContent/FormBuilder/FieldSet/FieldSet.jsx

```diff
@@ -33,23 +33,28 @@ let styles = theme => ({
     }
 });
 
-const FieldSetCmp = ({fieldset, classes, formik: {values, handleChange}}) => {
-    const isDynamicFieldSet = fieldset.dynamic;
-    const activatedFieldSet = (values && values[fieldset.name]) || !isDynamicFieldSet;
+const MemoizedFieldContainer = React.memo(FieldContainer);
 
+const DynamicFieldSet = ({fieldset, classes}) => {
```

Comment:
Split component to avoid having dependency on formik state on main component and whole subtree